classes/03class/exercises/c03-aws01/marcioantrielo/c03-aws01/c03-aws01.txt

```diff
@@ -0,0 +1,68 @@
+### Create an Auto Scaling Group 
+
+Details:
+
+  # How did you achieve that?
+	
+	AWS Console > EC2 > Auto Scaling > Auto Scaling Groups > Create Auto Scaling Group > Get Started > Amazon Linux 2 > t2.micro > Next: Configuration detail > 
+		
+		Name: devopsacademy-asg-01; 
+		> Skip Storage 
+		
+		Detail: 
+		Name:devopsacademy-asg-01
+		Purchasing option:On demand
+		EBS Optimized:No
+		Monitoring:No
+		IAM role:None
+		Tenancy:Shared tenancy (multi-tenant hardware)
+		Kernel ID:Use default
+		RAM Disk ID
+		Use default:User data
+		IP Address Type: Only assign a public IP address to instances launched in the default VPC and subnet. (default)
+		
+		> Create launch configuration 
+		> Select a key pair: devops-public 
+		
+		> Create Auto Scaling Group
+		> Group name: devopsacademy-asg
+		> Group size: 2
+		> Network: devopsacademy-vpc
+		> Subnet: public-b, public-c
+		> Scale between 1 and 2 instances
+		> Advanced Details	
+			User data:
+				#! /bin/bash
+				yum update -y
+				yum install -y httpd
+				curl 169.254.169.254/latest/meta-data/hostname > index.html
+				mv index.html /var/www/html/
+				systemctl start httpd
+		
+		> VPC 
+		> Elastic IPs
+		> Associate Address
+		
+   - Access through your browser each instance IP on port 80 http://<instance-ip> and post the hostname of each one
+	
+	- http://52.62.0.160
+	    ip-10-0-12-115.ap-southeast-2.compute.internal
+		
+	- http://52.65.96.70
+	    ip-10-0-14-105.ap-southeast-2.compute.internal
+
+
+   - Details of the security group used
+	
+	Group ID: sg-0d0904c0d9c82db0e
+	Group Name: devopsacademy-asg
+	VPC ID:  vpc-0c9124eda0e459981
+	Description: AutoScaling-Security-Group-1 (2020-03-15 21:41:11.999+11:00)
+	Owner: 650909960884
+	Inbound rule count: 3
```

Comment:
please detail the inbound and outbound rules, which is the core part of the Security Group.